v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/templates/ProtegrityDataTokenization.java

```diff
@@ -206,6 +208,11 @@ public static void main(String[] args) {
         .withValidation()
         .as(DataflowPipelineOptions.class);
 
+    if (DirectRunner.class.isAssignableFrom(options.getRunner())) {
+      String serviceAccount = nullToEmpty(dataflowOptions.getServiceAccount());
+      dataflowOptions.setServiceAccount(serviceAccount);
+    }
```

Comment:
I'm thinking - do we really need to check if it is a Direct runner specified? Probably, we can do something like this for simplicity:
```java
if (dataflowOptions.getServiceAccount() == null) {
  dataflowOptions.setServiceAccount("");
}
```

Reply:
We need have workaround about "org.apache.beam.runners.dataflow.options.DataflowPipelineOptions#getProject" option, that required for Dataflow runner, but not have sense for DirectRunner.

Reply:
Ok, but I'm afraid we will lost checking the service account for DataflowRunner when this parameter mustn't be null.

Reply:
You're right. The thing is that Dataflow takes care of Service Account itself, and we only need it to form a proper request to DSG. That's why we can simplify this 😁